src/gc/env/gcenv.os.h

```diff
@@ -87,7 +87,7 @@ class GCToOSInterface
     //  size    - size of the virtual memory range
     // Return:
     //  true if it has succeeded, false if it has failed
-    static bool VirtualRelease(void *address, size_t size);
+    static bool VirtualRelease(void *address);
```

Comment:
This change is bad. The VirtualRelease must have the size argument. While it is not used on CoreCLR and Windows, it is required for CoreRT on Unix.